js-miniapp-sdk/test/miniapp.spec.ts

```diff
@@ -173,3 +174,28 @@ describe('loadRewardedAd', () => {
     return expect(miniApp.loadRewardedAd(adUnitId)).to.eventually.equal(error);
   });
 });
+
+describe('shareInfo', () => {
+  it('should retrieve null from the MiniAppBridge once shareInfo call is successful', () => {
+    const sharedInfo = {
+      content: 'test content',
+    };
+    const response = null;
+
+    window.MiniAppBridge.shareInfo.resolves(response);
+    return expect(miniApp.shareInfo(sharedInfo)).to.eventually.equal(response);
+  });
+  it('should retrive error response from the MiniAppBridge once there is errors', () => {
+    const sharedInfo = {
+      content: 'test content',
+    };
+
+    const error: Error = {
+      message: 'Unknown error occured',
+      name: 'Bridge error',
+    };
```

Comment:
@minh-rakuten Is this the standard error format for all bridge communication ?? We are trying to follow a generic error format in iOS, something like below,
```
{
   "error":{
      "title":"ERROR_TITLE",
      "description":"DESCRIPTION OF THE ERROR"
   }
}
```

Reply:
Actually, the error format both android & bridge using is just the only string.

Reply:
I think we should use the standard [JavaScript error format](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error) - it uses `message` and `name` keys.

The [MDN docs for Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/reject) make it sound like using an instance of `Error` is the best practice.

Reply:
Sounds like we need to change on android, ios and bridge.
I will change the bridge code for `Error` in another PR.